sylvia-derive/src/message.rs

```diff
@@ -916,15 +917,52 @@ impl EntryPoints {
             )
             .unwrap_or_else(|| parse_quote! { #sylvia ::cw_std::StdError });
```

Comment:
This whole error finding is done the same for the `contract` macro. It could be extracted to some function to avoid duplication

Reply:
Yea, it probably could, but don't want to go into deep refactor with this one. It needs better overall structure.